authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/web/services/ConnectorService.kt

```diff
@@ -262,6 +263,7 @@ class ConnectorService(
             it.frontendUrl = connector.frontendUrl.trim()
             it.endpointUrl = connector.endpointUrl.trim()
             it.managementUrl = connector.managementUrl.trim()
+            it.jwksUrl = connector.jwksUrl?.trim()
```

Comment:
idea: can't you do common removing trailing slash and validation here?

Reply:
Actually, I want the validation to happen at the ApiService level instead of going that deep just to throw an error.
Moved the trimming to the trailing slash removal though